### PR TITLE
fix: headers update & warnings removed

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Warnings removed and headers added

## How did you test it?

Screenshot 2024-06-25 at 3 26 01 PM
Screenshot 2024-06-25 at 3 38 38 PM

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/Utilities/Utils.res [hunk 1]
```diff
@@ -799,11 +799,15 @@ let delay = timeOut => {
   })
 }
 let getHeaders = (~uri=?, ~token=?, ~headers=Dict.make(), ()) => {
+  let arrayOfNameAndVersion = String.split(Window.userAgent->OrcaLogger.browserDetect, "-")
   let headerObj =
     [
       ("Content-Type", "application/json"),
       ("X-Client-Version", Window.version),
       ("X-Payment-Confirm-Source", "sdk"),
+      ("Browser-Name", arrayOfNameAndVersion->Array.get(0)->Option.getOr("Others")),
```
Comment (ArushKapoorJuspay) [inline src/Utilities/Utils.res:808]: Please verify the names with @vsrivatsa-juspay

File: src/Utilities/Utils.res [hunk 2]
```diff
@@ -799,11 +799,15 @@ let delay = timeOut => {
   })
 }
 let getHeaders = (~uri=?, ~token=?, ~headers=Dict.make(), ()) => {
+  let arrayOfNameAndVersion = String.split(Window.userAgent->OrcaLogger.browserDetect, "-")
```
Comment (ArushKapoorJuspay) [inline src/Utilities/Utils.res:802]: We already have this function in orca-logger. Could you please use the same one

Comment (PritishBudhiraja) [reply src/Utilities/Utils.res:802]: Using that only.

File: src/Utilities/Utils.res (hunk 1 above)
Comment (PritishBudhiraja) [reply src/Utilities/Utils.res]: Updated

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by ArushKapoorJuspay — APPROVED** [review]

**Review by vsrivatsa-juspay — COMMENTED** [review]
What was the point of this?

**Review by vsrivatsa-juspay — APPROVED** [review]

**Pull request merged**